### PR TITLE
added ups query 

### PR DESCRIPTION
**Pull request by a-1an**
## What is this Pull Request About?

added ups search query, mentioned in issue #61 

## What will this Pull Request Affect?
will affect __init__.py file. 

# Any additional information?
---

### CODE REVIEW

File: src/searchor/__init__.py [hunk 1]
```diff
@@ -38,6 +38,7 @@ class Engine:
     Target = "https://www.target.com/s?searchTerm={query}"
     Textures = "https://www.textures.com/search?q={query}"
     Thesaurus = "https://www.thesaurus.com/browse/{query}"
+    Ups = "https://www.ups.com/us/en/SearchResults.page?q={query}&index=1"
```
Comment (ArjunSharda) [inline src/searchor/__init__.py:41]:
```suggestion
    Ups = "https://www.ups.com/us/en/SearchResults.page?q={query}"
```

**Review by ArjunSharda — CHANGES_REQUESTED** [review]
I think we can remove the &index=1 as it would work without it, and let's fully capitalize the UPS. Otherwise LGTM!

**Comment by ArjunSharda** [comment]
I've left some changes but otherwise perfect. Tell me when done and i'll merge

**Comment by a-1an** [comment]
> I think we can remove the &index=1 as it would work without it, and let's fully capitalize the UPS. Otherwise LGTM!

**Review by ArjunSharda — APPROVED** [review]
LGTM! Happy Hacktoberfest 2022!

**Pull request merged**